### ISSUE TITLE
Restart the project for kaios platform

### ISSUE THREAD

**Issue by altanik07** [szatkus/js2me#13]

**Comment by szatkus** [comment]
I already tried, but the simulator for KaiOS doesn't work well (especially for this app as it does some uncommon stuff) and I simply don't have means to test it.

Other than that, the project is open and I accept PRs.

**Comment by danimesq** [comment]
Duplicate of: https://github.com/szatkus/js2me/issues/7

**Comment by danimesq** [comment]
@altanik07, I don't think it should be a "restart". js2me can be multi-plaform while still supporting KaiOS.

> I already tried, but the simulator for KaiOS doesn't work well (especially for this app as it does some uncommon stuff) and I simply don't have means to test it.
> 
> Other than that, the project is open and I accept PRs.

I've replied here: https://github.com/szatkus/js2me/issues/7#issuecomment-1079769548 (tl;dr/positive signal: I do have a KaiOS device to test for you)

**Comment by danimesq** [comment]
@altanik07

I don't think it should be a "restart". js2me can be multi-plaform while still supporting KaiOS.

**Comment by szatkus** [comment]
If you want to port it I can support it as much as possible. I don't have any KaiOS device around, but, yeah, I can help you a bit.

**Comment by danimesq** [comment]
> If you want to port it I can support it as much as possible. I don't have any KaiOS device around, but, yeah, I can help you a bit.

Many thanks; I'll really need a hand in some things, probably.

But lets focus in the original issue: https://github.com/szatkus/js2me/issues/7

May I ask you or the OP to close issue #13 as duplicate?

**Comment by danimesq** [comment]
First, you replied only in a duplicate issue but ignored my comments in the original issue?

I'm still trying to understand what is happening.
